src/components/navigation/navigation/navigation.e2e.ts

```diff
@@ -54,7 +54,7 @@ describe('sbb-navigation', () => {
     expect(element).to.have.attribute('data-state', 'opened');
   });
 
-  it('sets the initial active actions and focuses on the first one', async () => {
+  it('sets the initial active actions', async () => {
```

Comment:
Could you add test about the reseting mechanism when section was closed and reopened again?